### PR TITLE
Updating js libs

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by ElDeveloper** [comment]
BTW, I just updated new-api, if you `git pull upstream new-api` it should all be kosher now.

**Comment by antgonza** [comment]
Ready for review.

**Comment by mortonjt** [comment]
First pass through.  So far so good. However, I did notice there are were quite a few images that were changed/deleted in the [vendor/css folder](https://github.com/biocore/emperor/tree/883d8aa9f905d125a27c745920a187fab5743857/emperor/support_files/vendor/css/images).  This is necessary right?

**Comment by antgonza** [comment]
Replaced all the required ones so my guess the missing ones were leftover ...

File: LICENSE.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Emperor makes usage of all of the following copyrighted packages; their licenses
 --------------------------------------------------------------------------------
 
 #### SlickGrid (v2.1.0-dev)
-[SlickGrid](https://github.com/mleibman/SlickGrid) - A lightning fast JavaScript grid/spreadsheet (e004912b5ce29ac0d0cb04df50fe66db5e3af9ea)
+[SlickGrid](https://github.com/6pac/SlickGrid) - A lightning fast JavaScript grid/spreadsheet (e004912b5ce29ac0d0cb04df50fe66db5e3af9ea)
```
Comment (ElDeveloper) [inline LICENSE.md]: We should probably update the SHA to point to d05dcd5b9369cae4b61df895ef7f51e8248e11c2 i.e. the SHA on @6pac's fork.

File: LICENSE.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ THE SOFTWARE.
 
 --------------------------------------------------------------------------------
 
-### THREE.js ([r68](https://github.com/mrdoob/three.js/tree/r68)) is released under the MIT License
+### THREE.js ([r73](https://github.com/mrdoob/three.js/tree/r68)) is released under the MIT License
```
Comment (ElDeveloper) [inline LICENSE.md]: Can you updated the URL to r73: https://github.com/mrdoob/three.js/tree/r73

File: LICENSE.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 --------------------------------------------------------------------------------
 
-### Spectrum ([1.0.2](https://github.com/bgrins/spectrum/tree/1.0.2)) is released under the MIT License
+### Spectrum ([1.8.0](https://github.com/bgrins/spectrum/tree/1.0.2)) is released under the MIT License
```
Comment (ElDeveloper) [inline LICENSE.md]: Can you update the URL to 1.8.0 https://github.com/bgrins/spectrum/tree/1.8.0 ?

File: emperor/support_files/vendor/css/slick.grid.min.css [hunk 4]
```diff
@@ -1 +1,158 @@
...
\ No newline at end of file
+/*
```
Comment (ElDeveloper) [inline emperor/support_files/vendor/css/slick.grid.min.css]: Can you minify this file? Also for some reason the file permissions changed from 644 to 755, if possible would be good to not have any of these be executable.

File: emperor/support_files/vendor/css/spectrum.css [hunk 5]
```diff
@@ -1,41 +1,47 @@
-
-
 /***
```
Comment (ElDeveloper) [inline emperor/support_files/vendor/css/spectrum.css]: Would you mind minifying this file?

File: emperor/support_files/vendor/js/three.js-plugins/ColorConverter.js [hunk 6]
```diff
@@ -8,6 +8,11 @@ THREE.ColorConverter = {
 	setHSV: function ( color, h, s, v ) {
```
Comment (ElDeveloper) [inline emperor/support_files/vendor/js/three.js-plugins/ColorConverter.js]: Would you mind changing the permissions to 644?

**Comment by ElDeveloper** [comment]
Looks good, thanks!! The only thing I noticed is that all the CSS files have an odd permission, would you mind changing this to 644? It seems like this is also the case for the following JS files: chosen.jquery.min.js, jquery-ui.min.js three.min.js, ColorConverter.js, OrbitControls.js, SVGRenderer.js &
Detector.js.

Aside from that it all looks good!

**Comment by ElDeveloper** [comment]
Yeah, the images are necessary, I think we may be able to prune down to
whatever we actually use but that may only be possible once we have
re-done the full app.

**Comment by antgonza** [comment]
Comments addressed ...

**Comment by ElDeveloper** [comment]
Thanks @antgonza!

**Pull request merged**